infrastructure/modules/ansible/roles/pyroscope/defaults/main.yml

```
@@ -0,0 +1,22 @@
+---
+pyroscope_version: "v0.0.21"
+pyroscope_host_port: "4040"
+pyroscope_host_path: "/pyroscope*"
+
+pyroscope_docker_network: "bridge"
+pyroscope_docker_image: "pyroscope/pyroscope"
+pyroscope_docker_tag: "{{ pyroscope_version }}"
+
+pyroscope_user_name: pyroscope
+pyroscope_user_group: pyroscope
+pyroscope_user_system_user: yes
+
+pyroscope_data_path: "/mnt/data/pyroscope"
+
+pyroscope_server_schema: "https"
```

I assume this should be scheme, not schema? :)

```suggestion
pyroscope_server_scheme: "https"
```